File: AA.CommoditiesDashboard.Api/AA.CommoditiesDashboard.Api/Model/CommodityResult.cs
using System;

namespace AA.CommoditiesDashboard.Api.Model
{
    public class CommodityResult
    {
        public DateTime Date { get; set; }

        public string Contract { get; set; }

        public decimal Price { get; set; }

        public int Position { get; set; }

        public int NewTradeAction { get; set; }

        public decimal PnlDaily { get; set; }

    }
}

File: AACommoditiesDashboardUi/src/app/models/commodity-result.ts
export class CommodityResult
{
    date: Date;
    contract: string;
    price: number;
    position: number;
    newTradeAction: number;
    pnlDaily: number;

    constructor(
        date: Date,
        contract: string,
        price: number,
        position: number,
        newTradeAction: number,
        pnlDaily: number
    )
    {
        this.date=date;
        this.contract=contract;
        this.price = price;
        this.position=position;
        this.newTradeAction=newTradeAction;
        this.pnlDaily=pnlDaily;
    }

}